### PR TITLE
#35432 Fix connection details title

### LINKED ISSUE #36432: Wrong connection details shell title
### Description

1. open connection configuration
2. Click on the Test Connection 
3. Click Details 
Expected shell name: **Properties for "connection-name"**
Actual : **Properties for "connection-url"**
{A378677C-D6B9-41C5-A3FD-4A8B0D50C50B}

{118BBF4B-CC72-4F40-B152-10183F4C004F}

### DBeaver Version

24.3.0

### Operating System

win10

### Database and driver

_No response_

### Steps to reproduce

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by IvanGorshechnikov**
Fix titlefor test connection properites

image

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/dialogs/connection/ConnectionWizard.java [hunk 1]
```diff
@@ -236,7 +236,7 @@ public void testConnection() {
 
                 new ConnectionTestDialog(
                     getShell(),
-                    targetDataSource,
+                    activeDataSource,
```
Comment (serge-rider) [inline plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/dialogs/connection/ConnectionWizard.java]: Why?

Comment (E1izabeth) [reply plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/dialogs/connection/ConnectionWizard.java]: +

Comment (IvanGorshechnikov) [reply plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/dialogs/connection/ConnectionWizard.java]: When a scheduling targetDataSource occurs, its Name is replaced from the table name to the full URL to the database

But in activeDataSource, the same name remains with the name of the table

**Comment by Matvey16** [comment]
Verified

**Pull request merged**